llvm/test/MC/AArch64/arm-poe2-tlbid-diagnostics.s

```diff
@@ -0,0 +1,88 @@
+// RUN: not llvm-mc -triple=aarch64 -show-encoding -mattr=+poe2 < %s 2>&1 \
+// RUN:        | FileCheck %s --check-prefix=CHECK-ERROR
+// RUN: not llvm-mc -triple=aarch64 -show-encoding -mattr=+poe2,+tlbid < %s 2>&1 \
+// RUN:        | FileCheck %s --check-prefix=CHECK-NO-REGISTER
+
+// Test without using +tlbid - no optional register operand allowed
+
+plbi alle3is, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle3os, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle2is, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle2os, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle1is, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle1os, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi vmalle1is, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi vmalle1os, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle3isnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle3osnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle2isnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle2osnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle1isnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi alle1osnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi vmalle1isnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+plbi vmalle1osnxs, x0
+// CHECK-ERROR: error: specified plbi op does not use a register
+
+
```

Comment:
Can you add a comment here:

// Tests where no optional register operand allowed

Reply:
Done.